### ISSUE TITLE
[FEATURE] Indication of private and public rooms

### ISSUE THREAD

**Issue by TekkadanPlays** [vicariousdrama/cornychat#73]
Being that rooms are stored via npub, and currently held in a single-file list, I think it would be most beneficial to users to offer an icon somewhere indicating whether the room is public or private. Users may not always remember to check the settings before opening up an old room, and they may not think to see if their room appears on the home page either. Having a visual cue within the room, especially a high contrast (such as black/white, or an icon of a ghost/ostrich to indicate private or public, respectively) may help users from accidentally lingering within a private room.

This same indicator could also be referenced under 'My Rooms'.

If the user has visited a private room, these rooms could also be listed on the main page, beneath open rooms, if they are currently open. This indicator, in conjunction with a separated section, would give a clear indication of which private rooms are currently available, and give the user a more refined sense of when a room is private or public.

An idea for the UI element of this could be an open/closed eyeball in a recognizable position amongst various elements such as:

- listed rooms (on both main page and under 'my rooms')
- title bar of each room
- separate section beneath public rooms for private rooms (indicator icon could be used in the header of both public and private rooms, to assist the user in translating the icon to its meaning quickly)
- could also offer a separate section under 'my rooms' for private rooms, following the above theme

Example icons:

![240_F_302717760_ocCYWSkmvCrK4h1OZ5Yoxz60BkZ5fopJ](https://github.com/vicariousdrama/cornychat/assets/93434084/f232c7d0-5d53-4053-b66b-76d5c47c465c)

**Comment by vicariousdrama** [comment]
I agree with these sentiments.  Here are some things I want to do to address this, though I haven't yet done the design work or figured out appropriate icons

- List of Live Rooms
  - [ ] Indicate whether room requires a passphrase for entry
  - [ ] Indicate whether room is announcing to nostr as a live activity
- List of My Rooms
  - [ ] Indicate whether room requires a passphrase for entry
  - [ ] Indicate if room is marked as public or private
- List of Scheduled Events 
  - [ ] Indicate whether room requires a passphrase for entry
  - [ ] Indicate room owner
- In room display
  - [ ] Redesign the header bar, getting rid of the padding around components
  - [ ] Remove the room logo image from the top left corner
  - [ ] When clicking avatar (currently in top right) that lists number of users in room, present a popup showing other active rooms and the people in them
  - [ ] Move Room Tips to the right side with other indicators
  - [ ] Consolidate the Room Tip action and the v4v on/off indicator
  - [ ] Add indicators for room being public/private, closed/open, passphrase required or not, publishing live activity
  - [ ] Some indicators are clickables (links, tips, users) while others aren't. Need to take care to separate the clickable ones enough to avoid fat fingering an unintended one
  - [ ] Move "Leave Room" on nav bar to the three dot menu
  - [ ] Change three dot menu to standard hamburger menu icon
  - [ ] Change menu list to be one entry per line, accordion upwards
- Room Settings
  - [ ] Show a rendering of the room logo uri in room settings similar to what is done for slides
  - [ ] Show a rendering of room background uri in room settings similar to what is done for slides

The notion of listing private rooms in the live room list seems like a useful one for people that were in that room recently, but I'll need to sort out a good way to discover without incorrectly exposing private rooms to others.
